src/main/java/logic/RollLogic.java

```diff
@@ -121,9 +127,9 @@ private static EmbedBuilder postProcessResult(String dicePool, User user, EmbedB
      */
     private static Try<String> convertSkillsToDice(User user, String pool) {
         // We use join to get access to CompletionException, which would allow us to handle the exceptions thrown in the CompletableFuture
-        //noinspection unchecked
-        final Try<Map<String, String>> skillMap = Try.of(() -> SheetsHandler.getSkills(user).get()).mapFailure(API.Case($(t -> t instanceof ExecutionException), Throwable::getCause));
-        return skillMap.map(map -> Arrays.stream(pool.split(" ")).map(s -> map.getOrDefault(s, s)).collect(Collectors.joining(" ")));
+        return Try.of(() -> SheetsHandler.getSkills(user).get())
+                .recoverWith(throwable -> throwable instanceof IllegalArgumentException ? Try.success(new HashMap<>()) : Try.failure(throwable))
```

Comment:
I'd throw a specific "UserNotFound" or something is that's the error you're going for.  It is also entirely value to handle no user as just having the SheetsHandler returning Successful for nothing.